kubernetes/values.yaml

```diff
@@ -0,0 +1,48 @@
+hub:
+  config:
+    Authenticator:
+      admin_users:
+        - yfw215
+        - demouser
+    DummyAuthenticator:
+      password: 'alpha_automl'
+    JupyterHub:
+      admin_access: true
+      authenticator_class: tmpauthenticator.TmpAuthenticator
+  shutdownOnLogout: true
+  networkPolicy:
+    enabled: false
+proxy:
+  service:
+    type: ClusterIP
+  chp:
+    networkPolicy:
+      enabled: false
+  traefik:
+    networkPolicy:
+      enabled: false
+singleuser:
+  storage:
+    capacity: 10Gi
+  networkPolicy:
+    enabled: false
+  cloudMetadata:
+    blockWithIptables: false
+  image:
+    name: edenwu/alpha_automl
```

Comment:
Now that an initial image was pushed, we can modify this to point to the image at `ghcr.io/vida-nyu/alpha-automl`.